nistats/reporting/compare_niimgs.py

```diff
@@ -0,0 +1,93 @@
+import os
+import warnings
+
+import numpy as np
+from scipy import stats
+from matplotlib import pyplot as plt
+
+
+def compare_niimgs(ref_imgs, src_imgs, masker, plot_hist=True, log=True,
+                   ref_label="image set 1", src_label="image set 2",
+                   output_dir=None, axes=None):
+    """Creates plots to compare two lists of images and measure correlation.
+
+    The first plot displays linear correlation between voxel values
+    The second plot superimposes histograms to compare values distribution
+
+    Parameters
+    ----------
+    ref_imgs: nifti_like
+        reference images.
+
+    src_imgs: nifti_like
+        Source images.
+
+    log: Boolean, optional (default True)
+        Passed to plt.hist
+
+    plot_hist: Boolean, optional (default True)
+        If True then histograms of each img in ref_imgs will be plotted
+        along-side the histogram of the corresponding image in src_imgs
+
+    ref_label: str
+        name of reference images
+
+    src_label: str
+        name of source images
+
+    output_dir: string, optional (default None)
+        Directory where plotted figures will be stored.
+
+    axes: list of two matplotlib Axes objects, optional (default None)
+        Can receive a list of the form [ax1, ax2] to render the plots.
+        By default new axes will be created
+
+    Returns
+    -------
+    Pearsonr correlation between the images
+
+    Examples
+    --------
+    [1] check_zscores.compare_niimgs(["/home/elvis/Downloads/zstat2.nii.gz"],
```

Comment:
One should create an actual example for that one and remove this from the docstring.

Reply:
And this is actually completely unrelated to that PR.

Reply:
Yeah we are leaving that in there. compare_niimgs may be unrelated but we have to refactor reporting.py so this is part of that.